google/resumable_media/_upload.py

```diff
@@ -817,7 +817,7 @@ def get_next_chunk(stream, chunk_size, total_bytes):
             raise ValueError(msg)
 
     content_range = get_content_range(start_byte, end_byte, total_bytes)
-    return start_byte, end_byte, payload, content_range
+    return start_byte, max([end_byte, 0]), payload, content_range
```

Comment:
@lukesneeringer This feels a little hacky. Let me dig in to the source and see if there is a better place for it.